common/lib/dependabot/pull_request_creator/branch_namer.rb

```diff
@@ -165,12 +165,12 @@ def new_library_requirement(dependency)
         updated_reqs.first[:requirement]
       end
 
-      # TODO: Look into bringing this in line with existing library checks that
-      # we do in the update checkers, which are also overriden by passing an
-      # explicit `requirements_update_strategy`.
+      # TODO: Bring this in line with existing library checks that we do in the
+      # update checkers, which are also overriden by passing an explicit
+      # `requirements_update_strategy`.
+      #
+      # TODO re-use in MessageBuilder
       def library?
-        return true if files.map(&:name).any? { |nm| nm.end_with?(".gemspec") }
```

Comment:
Not 💯  sure about this yet, might be better to only use the requirement if it's changed instead if inferring change based on this